python/cudf/cudf/core/dataframe.py

```diff
@@ -2123,6 +2133,9 @@ def loc(self):
     def iloc(self):
         """
         Selecting rows and column by position.
+        If DataFrame is of mixed types and result is expected
```

Comment:
I think something like this better fits in a `Notes` section rather than the top level description. According to https://numpydoc.readthedocs.io/en/latest/format.html, the `Notes` section goes after the `See also` section.

Would suggest something like this:

```
Notes
------
One notable difference from Pandas is when returning a single row...

<example here> 
```